src/main/java/org/opensearch/geospatial/ip2geo/action/DeleteDatasourceRequest.java

```diff
@@ -43,9 +45,9 @@ public DeleteDatasourceRequest(final StreamInput in) throws IOException {
     @Override
     public ActionRequestValidationException validate() {
         ActionRequestValidationException errors = null;
-        if (name == null || name.isBlank()) {
+        if (VALIDATOR.validateDatasourceName(name) != null) {
```

Comment:
nit: you can use Objects.nonNull()

Reply:
Thanks. I feel `!= null` is more readable. Anyway, this code will be removed with update.